iati-organisations-schema.xsd

```diff
@@ -49,7 +49,7 @@
           <xsd:documentation xml:lang="en">
             A date/time stamp for when this file was generated. This
             is not necessarily the last-updated date for the
-            individual activity records in it. Use of this attribute
+            individual organisation records in it. Use of this attribute
```

Comment:
Also agree.  There can be multiple organisations in any organisation file, and this timestamp relates to them